src/OrchardCore.Modules/OrchardCore.Workflows/Http/Scripting/HttpMethodsProvider.cs

```diff
@@ -129,21 +129,18 @@ public HttpMethodsProvider(IHttpContextAccessor httpContextAccessor)
                 })
             };
 
+            // This should be deprecated
             _queryStringAsJsonMethod = new GlobalMethod
             {
                 Name = "queryStringAsJson",
-                Method = serviceProvider => (Func<JObject>)(() =>
-                    new JObject((from param in httpContextAccessor.HttpContext.Request.Query
-                                 select new JProperty(param.Key, JArray.FromObject(param.Value.ToArray()))).ToArray()))
+                Method = serviceProvider => _deserializeRequestDataMethod.Method.Invoke(serviceProvider)
             };
 
             // This should be deprecated
             _requestFormAsJsonMethod = new GlobalMethod
             {
                 Name = "requestFormAsJson",
-                Method = serviceProvider => (Func<JObject>)(() =>
-                    new JObject((from field in httpContextAccessor.HttpContext.Request.Form
-                                 select new JProperty(field.Key, JArray.FromObject(field.Value.ToArray()))).ToArray()))
+                Method = serviceProvider => _deserializeRequestDataMethod.Method.Invoke(serviceProvider)
```

Comment:
This will change the format of the data, which will break existing apps that use it.